Automattic/vip-go-mu-plugins
# Update Jetpack 10.8 tag to beta2

## Description
Previously, the tag was at `10.8-beta`. This updates it to `10.8-beta2`.

## Changelog Description

### Plugin Updated: Jetpack 10.8

We upgraded Jetpack 10.8-beta to Jetpack 10.8-beta2.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. [Pin Jetpack to 10.8](https://docs.wpvip.com/how-tos/test-jetpack-on-vip-go/#h-pinning-to-a-version)
3. Go to wp-admin > Jetpack
4. Verify Jetpack version is 10.8-beta2
